app/controllers/memberships_controller.rb

```diff
@@ -0,0 +1,38 @@
+class MembershipsController < ApplicationController
+
+  def join
+    @organization = Organization.find membership_params[:id]
+    membership = @organization.memberships.new(user: current_user)
+
+    if membership.save
+      flash[:success] = "Membership request sent."
+    else
+      flash[:danger] = membership.errors.full_messages
+    end
+
+    redirect_to @organization
+  end
+
+  def deactivate
+    @membership = Membership.find(membership_params[:membership_id])
```

Comment:
Don't need this to be an instance variable if we're not rendering it in the template later.